app/controllers/api/v0/api_controller.rb

```diff
@@ -51,8 +51,8 @@ def authorize_super_admin
       #
       # @return [User, NilClass]
       #
-      # @see {#pundit_user} for one way we use this method
```

Comment:
I don't have enough context here - was this an older supported style that's warning now, or were we using a tool different from yard to extract documentation from the code?